debug-web-ui/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
I think this defaults are from create-react-app and they may conflict with bootstrap's defaults (not sure)